resources/operator.yaml

```diff
@@ -28,5 +28,23 @@ spec:
               valueFrom:
                 fieldRef:
                   fieldPath: metadata.name
+            - name: ANSIBLE_VERBOSITY
```

Comment:
Do we want to just add this to the normal definition of operator? We've another definition in `operator_store_results` which allows for indexing uperf/fio(pr_pending) results too. Should we look to add this work into that definition as well?

Reply:
I don't see why not? Since we build our operator, I would prefer we have everything built...